### PR TITLE
SEO-178107-JS-Diagram-Changes

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK,

Worked on page with redirect for js diagram.

Regards,
Sharly A.

### CODE REVIEW

File: ej2-javascript/diagram/js/data-binding.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ To map the user defined JSON data with diagram, configure the fields of [`dataSo
 
 You can bind the diagram with remote data by using [`dataManager`].
 
-It uses two different classes: `DataManager` for processing and `Query` for serving data. `DataManager` communicates with data source and `Query` generates data queries that are read by the [`dataManager`](../api/diagram/dataSourceModel).
+It uses two different classes: `DataManager` for processing and `Query` for serving data. `DataManager` communicates with data source and `Query` generates data queries that are read by the [`dataManager`](../api/diagram/dataSourceModel/).
 
 To learn more about data manager, refer to [`Data Manager`](../api/diagram/dataSourceModel).
```
Comment (Yvone-Atieno) [inline ej2-javascript/diagram/js/data-binding.md:56]: Check the above link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-javascript/diagram/js/data-binding.md (hunk 1 above)
Comment (Sharly-Atieno) [reply ej2-javascript/diagram/js/data-binding.md]: Updated the link

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**